### ISSUE TITLE
Still exclude folders even with empty Exclusion filter/Ignore list

### ISSUE THREAD

**Issue by bphd** [arsenetar/dupeguru#1068]

**Comment by arsenetar** [comment]
This issue requires additional information on what was expected and what the behavior was, without additional information this is not reproducible.

**Comment by bphd** [comment]
Reproduction: Emptying list of Exclusion filter/Ignore list, add a folder

expected: no red
seen: full red

**Comment by wanderr** [comment]
I experienced this as well. Added .*workspace.* to my exclusion list. Then I removed it, but everything matching the regex was still excluded

**Comment by bphd** [comment]
Yeah it astound me that something that I was able to reproduce since I know this software, on any computer and any OSes, something that big, a "core" function that widely broken still need additional information, like it didn't jump to the eyes that the filter function just... Doesn't work at all? I need to prove it like the very own dev don't use the software they're supposed to work on day long

But anyway, if people want just fancily written forms instead of actually reworking the code, I mean, it's a modern developer era problem
